src/reducers/books.js

```diff
@@ -0,0 +1,25 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/types';
+
+const initialState = [
```

Comment:
According to the instructions on pathwright, I think you need to move this to `src/index.js`

![image](https://user-images.githubusercontent.com/31897434/122143582-a6b8de80-ce49-11eb-86a3-ffbf2e5e22d4.png)

[Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376028/)